docs/man-pages/condor_vacate_job.rst

```diff
@@ -1,119 +1,119 @@
+    Display version information.
+ **-long**
+    Display result ClassAd.
+ **-totals**
+    Display success/failure totals.
+ **-fast**
+    Perform a fast vacate and hard kill the jobs.
+ **-pool** *hostname[:portnumber]*
     Specify a pool by giving the central manager's host name and an
-    optional port number
+    optional port number.
  **-name** *scheddname*
-    Send the command to a machine identified by *scheddname*
+    Send the command to a machine identified by *scheddname*.
  **-addr** *"<a.b.c.d:port>"*
-    Send the command to a machine located at *"<a.b.c.d:port>"*
+    Send the command to a machine located at *"<a.b.c.d:port>"*.
+ **-constraint** *expression*
+    Vacate all jobs which match the job ClassAd expression constraint.
+ **-all**
+    Vacate all the jobs in the queue.
  *cluster*
-    Vacate all jobs in the specified cluster
+    Vacate all jobs in the specified cluster.
  *cluster.process*
-    Vacate the specific job in the cluster
+    Vacate the specific job in the cluster.
  *user*
-    Vacate jobs belonging to specified user
- **-constraint** *expression*
-    Vacate all jobs which match the job ClassAd expression constraint
- **-all**
-    Vacate all the jobs in the queue
- **-fast**
-    Perform a fast vacate and hard kill the jobs
+    Vacate jobs belonging to specified user.
 
 General Remarks
 ---------------
 
-Do not confuse *condor_vacate_job* with *condor_vacate*.
-*condor_vacate* is given a list of hosts to vacate, regardless of what
-jobs happen to be running on them. Only machine owners and
-administrators have permission to use *condor_vacate* to evict jobs
-from a given host. *condor_vacate_job* is given a list of job to
-vacate, regardless of which hosts they happen to be running on. Only the
-owner of the jobs or queue super users have permission to use
-*condor_vacate_job*.
+If the **-name** option is specified, the named *condor_schedd* is targeted
+for processing. Otherwise, the local *condor_schedd* is targeted.
+
+Vacated jobs will be sent a soft kill signal (``SIGTERM`` by default or
+the value of :ad-attr:`KillSig` in the job's ClassAd) unless **-fast** is used.
+
+Using *condor_vacate_job* on jobs which are not currently running has
+no effect.
+
+.. warning::
+
+    Do not confuse this tool with :tool:`condor_vacate`. :tool:`condor_vacate`
+    is intended for use by the owners/administrators of an Execution Point to
+    evict jobs from their host machine.
+
+Exit Status
+-----------
+
+0  -  Success
+
+1  -  Failure has occurred
 
 Examples
 --------
 
-To vacate job 23.0:
+To vacate a specific:
 
 .. code-block:: console
 
     $ condor_vacate_job 23.0
 
-To vacate all jobs of a user named Mary:
+To vacate a job fast:
+
+.. code-block:: console
+
+    $ condor_vacate_job -fast 23.0
+
+To vacate all jobs owned by user Mary:
 
 .. code-block:: console
 
-    $ condor_vacate_job mary
+    # condor_vacate_job mary
 
 To vacate all vanilla universe jobs owned by Mary:
```

Comment:
What does `condor_vacate_job` do on a scheduler or local universe job?  Should we mention that explicitly, as that might be a surprise?